### ISSUE TITLE
Greek - Phi Φ - spacing feel loose generally in all roman weights

### ISSUE THREAD

**Issue by EbenSorkin** [TypeNetwork/Roboto-Extremo#403]
Greek - Phi Φ - spacing feel loose generally in all roman weights

<img width="1494" alt="image" src="https://github.com/googlefonts/roboto-flex/assets/2181213/99552d93-e26a-4fe8-b8ad-ccee78fdd360">
 Note the 6th style which I am using to compare is ( lower right ) is Commisioner by Kostas Bartsokas

**Comment by dberlow** [comment]
Eben,

We do not have sources for opsz0 and do not know what the "Hard-Wrap" axes does. This is important when posting an issue comparing a font with no optical size defined to a mystery location in our font. 

I also see from another issue, that the spacing you show is not identified for design space and is not accurate to the sources we provided. 

Can you point to the branch where this font, and these styles are being made?

What is the design space of Commisioner? and where is the specimen you are showing in that design space?

**Comment by EbenSorkin** [comment]
I used the latest build from Marc which was about a week old at the time. I
can be more specific about where in the space this is but for now it’s
default with only weight variation. Commissioner is useful because it is a
Greek designed by a Greek person with strong type design skills.

Should I use a different source?

**Comment by davelab6** [comment]
> We do not have sources for opsz0 

Right, but the way CSS works, when you specify an axis value beyond the axis range, it will clamp to the nearest value that the axis does have. So opsz 0 is a short hand to get opsz min.

> and do not know what the "Hard-Wrap" axes does.

This is a possible UX improvement for the https://github.com/vikagrabowska/Octopus web app, but this (iirc) is not an axis setting, but a setting for how many characters to wrap at.

> This is important when posting an issue comparing a font with no optical size defined to a mystery location in our font.

I agree that the full instance location should be specified in all issues. The image shows 2 other axes locations, weight 770 and width 100. I was going to ask, Eben, were any other axes set? but you already replied to say only weight.

So there's a UX improvement for Octopus to suggest too, the Show > Parameters feature of https://typetools.typenetwork.com/ :) 

> I also see from another issue, that the spacing you show is not identified for design space and is not accurate to the sources we provided.

I'm curious how you see this? Seems default spacing for opsz 8 wght 770 to me from a quick look :)

>> Can you point to the branch where this font, and these styles are being made?
>
> I used the latest build from Marc which was about a week old at the time

Eben, please be very specific; I think its important to use the latest file in this repo (from https://github.com/googlefonts/roboto-flex/tree/main/fonts) not some random version on you HD. 

> What is the design space of Commisioner? and where is the specimen you are showing in that design space?

https://github.com/kosbarts/Commissioner README tells the design space; I agree knowing its instance location and any other settings is pertinent.

**Comment by dberlow** [comment]
So, as the opsz min of this is opsz8 it should be reviewed at that size and if compared to another font, to it’s 8 point of the same weight at actual size. In addition: Every single glyph in the variable font this is a subset of, has multiple parametric axes that can be blended for whatever contrast or weight, or width is desired.  And: Every glyph in the font is properly spaced by a Greek designer and critiqued by another designer familiar with Greek, and the spacing of the sources in the main directory, as prepared by FB, are properly spaced and not offset to one side or the other.First the MainsThen the duovars.They are all centered, contrary to the proofs I was shown.Cheers, David
